packages/design-tokens/src/scss/themes/light-inverted/_shadows.scss

```diff
@@ -1,13 +1,16 @@
 // Generated Shadows from Supernova. Do not edit manually.
-$focus: 0 0 0 2px #d2c2ff99 !default;
+// Copied from light theme
 $shadow-100: 0 2px 8px 0 #00000026 !default;
 $shadow-200: 0 4px 12px 0 #0003 !default;
 $shadow-300: 0 8px 24px 0 #00000040 !default;
 $shadow-400: 0 12px 32px 0 #00000040 !default;
 
+$focus: 0 0 0 2px #d2c2ff99 !default;
```

Comment:
Not reusing spacing and color tokens here?

Reply:
We can check if figma allows this, otherwise I think about them as tokens on the same level and then it might feel weird to somehow join them.